app/views/publishers/schools/_organisation.html.slim

```diff
@@ -1,48 +1,38 @@
 = govuk_summary_list do |summary_list|
   - summary_list.row do |row|
-    - row.key text: t("schools.description")
-    - row.value text: organisation.description? ? truncate(organisation.description, length: 130) : "Not provided"
-    - row.action text: t("buttons.change"), href: edit_publishers_school_path(organisation), classes: "govuk-link--no-visited-state"
+    - row.key text: t(".name.label")
+    - row.value text: organisation.name
 
   - unless organisation.local_authority?
     - summary_list.row do |row|
-      - row.key text: t("schools.address")
+      - row.key text: t(".address.label")
       - row.value text: full_address(organisation)
 
   - if organisation.school?
     - summary_list.row do |row|
-      - row.key text: t("schools.school_age")
-      - row.value text: age_range(organisation)
-
-  - if organisation.school?
-    - summary_list.row do |row|
-      - row.key text: t("schools.type")
+      - row.key text: t(".type.label")
       - row.value text: organisation.school_type
-  - elsif organisation.trust?
-    - summary_list.row do |row|
-      - row.key text: t("school_groups.group_type")
-      - row.value text: organisation.group_type
 
-  - if organisation.school?
-    - summary_list.row do |row|
-      - row.key text: t("schools.website")
-      - row.value text: organisation.url ? open_in_new_tab_link_to(organisation.url, organisation.url, class: "wordwrap") : t("jobs.not_defined")
-      - row.action text: t("buttons.change"), href: edit_publishers_school_path(organisation), classes: "govuk-link--no-visited-state"
-  - else
-    - summary_list.row do |row|
-      - row.key text: t("school_groups.website", organisation_type: organisation_type_basic(organisation).humanize)
-      - row.value text: organisation.url ? open_in_new_tab_link_to(organisation.url, organisation.url, class: "wordwrap") : t("jobs.not_defined")
-      - row.action text: t("buttons.change"), href: edit_publishers_school_path(organisation), classes: "govuk-link--no-visited-state"
+    - if organisation.phase.present?
+      - summary_list.row do |row|
+        - row.key text: t(".phase.label")
+        - row.value text: t(".phase.values.#{organisation.phase}")
 
-  - if organisation.school?
-    - summary_list.row do |row|
-      - row.key text: t("schools.urn")
-      - row.value text: organisation.urn
-  - elsif organisation.trust?
     - summary_list.row do |row|
-      - row.key text: t("school_groups.uid")
-      - row.value text: organisation.uid
+      - row.key text: t(".school_age.label")
+      - row.value text: age_range(organisation)
+
+    - if (size = organisation.gias_data["NumberOfPupils"].presence)
```

Comment:
Assuming we can use `school_size(organisation)` here.

Can be found in:

`app/helpers/organisations_helper.rb`

Reply:
Rather than not showing this row if we don't have this data, should we also show "No information available" as we do here if we don't have this info:

`app/views/vacancies/listing/_school.html.slim`

Reply:
Good question, probably need to confirm with prod - the ticket states not to display education phase if the information is missing, so I want with the same logic here.

Actually it seems there's even a better helper there - `number_of_pupils(school)` :)

Reply:
It is now confirm that we should not display any information that is not provided from gias and users can't add.